docs/contributing-to-airbyte/resources/qa-checks.md

```diff
@@ -27,6 +27,33 @@ _Applies to connector with any Airbyte usage level_
 
 The user facing connector documentation should be stored under `./docs/integrations/<connector-type>s/<connector-name>.md`.
 
+### Connectors documentation headers structure follows our guidelines
+
+_Applies to the following connector types: source_
+_Applies to the following connector languages: python, low-code_
+_Applies to connector with any support level_
+_Applies to connector with any Airbyte usage level_
+
+The user facing connector documentation should follow the guidelines defined in the [documentation standards](https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw).
```

Comment:
As:
* the check is pretty explicit
* This `hackmd.io` link is not maintained (I think so)
* The `qa-checks.md` is considered as the source of truth for what it takes to have a "clean" connector

**I would highly recommend explicitly stating in the check descripition what is the expected header structure.**

Same applies for `CheckDocumentationDescriptions`.

Reply:
Updated descriptions for check and added examples from template.md as "source of truth"